elastalert/elastalert.py

```diff
@@ -994,6 +1010,25 @@ def start(self):
             sleep_duration = total_seconds(next_run - datetime.datetime.utcnow())
             self.sleep_for(sleep_duration)
 
+    def wait_until_responsive(self, timeout=None, clock=timeit.default_timer):
+        """Wait until ElasticSearch becomes responsive (or too much time passes)."""
+
+        # Elapsed time is a floating point number of seconds.
+        timeout = timeout.total_seconds()
+
+        # Periodically poll ElasticSearch.  Keep going until ElasticSearch is
+        # responsive *and* the writeback index exists.
+        ref = clock()
+        while (timeout is None) or ((clock() - ref) < timeout):
+            try:
+                if self.writeback_es.indices.exists(self.writeback_index):
```

Comment:
BTW, there is currently not possibility to end up with `timeout is None` so I will remove it (there will never be an infinite loop).